python/tvm/relay/frontend/onnx.py

```diff
@@ -3176,7 +3176,7 @@ def _impl_v10(cls, inputs, attr, params):
     @classmethod
     def _impl_v13(cls, inputs, attr, params):
         data, scale, zp = inputs
-        axis = attr.get("axis", 1)
+        axis = attr.get("axis", 0)
```

Comment:
I don't think this is correct. The default in the ONNX spec is 1. I think we should leave it as 1, I think there will be cases that fail with this change, but have code to check for data with 0D or 1D shapes and set it to 0 in that case?

https://github.com/onnx/onnx/blob/master/docs/Changelog.md#DequantizeLinear-13

Reply:
yeah confirmed docs are correct, here is where the default is set: https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/providers/cpu/tensor/quantize_linear.h#L17

Reply:
I've also hit this issue when I was testing a quantized tflite model, the change below worked (as @mbrookhart suggested)
https://github.com/masahi/tvm/blob/e649837fb4f74abbbb770e0a3f6bf175f407cc7d/python/tvm/relay/frontend/onnx.py#L3199-L3202